07LessonSeven/homework/css/style.css

```diff
@@ -0,0 +1,82 @@
+h1 {
+  padding-left: 268px;
+}
+
+ul {
+  list-style-type: none;
+  background-color: #313131;
+  display: flex;
+  padding-left: 256px;
+  height: 80px;
+  align-items: center;
+  color: white;
+}
+
+#box-left {
```

Comment:
Try not to use ID selectors if you can avoid it. Use element electors and classes instead. Remember ID selectors have a very high specificity (remember the shark in the info graphic? review http://bit.ly/2aUlMEU). Make it easy for your future self so you can override CSS properties easily by initially using a selector that has a low specificity before you use one that has a high specificity (ID selectors a.k.a The Shark eats fish who eats plankton, see the graphic as it will make sense) :smiley: